### PR TITLE
Implement page with a list of experiments

### PR DESCRIPTION
**Pull request by smacker**
Implements the first point in https://github.com/src-d/code-annotation/issues/149

screen shot 2018-02-21 at 17 24 48

The page is available by url `/dashboard`.

### CODE REVIEW

File: src/components/ExperimentsList.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Table, Button } from 'react-bootstrap';
+import './ExperimentsList.less';
+
+function buttonText(procent) {
```
Comment (carlosms) [inline src/components/ExperimentsList.js]: percent?

File: src/pages/Experiments.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Grid, Row, Col } from 'react-bootstrap';
+import { Helmet } from 'react-helmet';
+import PageHeader from '../components/PageHeader';
+import ExperimentsList from '../components/ExperimentsList';
+
+class Final extends Component {
```
Comment (carlosms) [inline src/pages/Experiments.js]: should this be class Experiments?

File: src/components/ExperimentsList.js (hunk 1 above)
Comment (smacker) [reply src/components/ExperimentsList.js]: pardon my Russian. fixed.

File: src/pages/Experiments.js (hunk 2 above)
Comment (smacker) [reply src/pages/Experiments.js]: missed it. Thanks! Fixed.

**Review by carlosms — APPROVED** [review]

**Review by bzz — APPROVED** [review]

**Pull request merged**